### DISCUSSION TITLE
erm.. ples stop removing features every update

### DISCUSSION THREAD

**Discussion by scanash00** [zen-browser/desktop#1657] [General]
Updates are supposed to add new things to the browser, the installer also shows a crash thing when updating, why on hearth we can't create a damn tab by pressing a button now, i do not want to do ctrl t to open a new tab.. GET OUT

**Comment by scanash00** [comment]
i also hate there are 3000 buttons in the top bar. it was good before

**Comment by scanash00** [comment]
Good that they added a button to umm. new tab.

but atleast make it so there is less buttons on the top bar

**Comment by nocanoa** [comment]
Its in alpha things change for the better and sometimes for the worst, but that's why we got updateds to fix shit we f up and add new stuff, nobody is perfect.
